contributions/Dockerfile

```diff
@@ -23,24 +23,16 @@ WORKDIR /usr/src/app/contributions/api
 
 COPY contributions/contribution.yaml .
 
-ENV MONGO_CONTRIBUTION_URL='' \
-    API_LOC='../' \
+ENV API_LOC='../' \
```

Comment:
Let's use the default options so that the docker container will run with minimum environment variables. Also, FIELD_OBJECT_ID and FIELD_NAME can be removed as these are not environment variables.
```
ENV API_LOC='../' \
    MONGO_CONTRIBUTION_URL='localhost:27017' \
    FLASK_APP='contributions_rest_service' \
    FLASK_ENV='production' \
    CONTRIBUTION_URL_PREFIX='' \
    CONTRIBUTION_DB_NAME='contributions_db' \
    CONTRIBUTION_COLL_NAME='contributions' \
    DEBUG='False' \
    CORS_ENABLED='False'
```

Reply:
Okay. Changed.

Reply:
Hi @ywkim312, there are some default values that we could use in the previous code block. Please see. Thanks.

Reply:
Added default values.

Reply:
Thanks.